content/shmem_quiet.tex

```diff
@@ -22,7 +22,7 @@
 \end{apiarguments}
 
 \apidescription{ 
-    The \FUNC{shmem\_quiet} routine ensures completion of \PUT{}, \ac{AMO},
+    The \FUNC{shmem\_quiet} routine ensures completion of \PUT{}, strided \PUT{}, \ac{AMO},
```

Comment:
I think the general feedback was to make a broader change to define Put operations as block put, elemental put, and strided put to simplify this sentence and related sentences throughout the text.